### DISCUSSION TITLE
Talent Loadout Conversion Format

### DISCUSSION THREAD

**Discussion by Jed-Giblin** [TobiasM95/WoW-Talent-Tree-Manager#22] [Q&A]
Do you have any docs / soruces on how the talent tree gets converted into the "Loadout Code" format used for sharing? I would like to write a small script to parse a list of talents and turn it into a wowhead talent calculator

**Comment by TobiasM95** [comment]
Hey, sorry for the late reply.
Are you asking about the 
- TTM talent tree export string
- TTM loadout export string
- TTM skillset export string
- Base64 Blizzard skillset export string?
All of these are included in the code in the repo (I think in TalentTrees.cpp/h). I can answer more specifics of how they're generated but most of the talent stuff on TTM's side is basically a list of talents with 0 for not selected, 1/2 for switch talents and N for N points in the talent for passives.
Wowhead uses another form of hash function for their URL updates which I don't know how it works yet.
